### PR TITLE
Added Header Bar and Forum Box Component

### PR DESCRIPTION
**Pull request by AbhiKul10**
**Please provide enough information so that others can review your pull request:**

Added Header Bar and Forum Box Component

**Explain the **details** for making this change. What existing problem does the pull request solve?**

- HeaderBar is a customized Header bar serving UI enhancement purposes.
- ForumBox can be used as a customized container, having heading and TextInput.

### CODE REVIEW

File: app/components/ForumBox/ForumBox.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <View style={{marginTop: 15}}>
+                
+                <View style = {styles.QuestionStyle}>
+                    <View style={styles.nameHeadlineStyle}>
+                        <Text style={{marginLeft: 10}}>{this.props.Question}</Text>
+                    </View>
+                    <View style={{backgroundColor:'white', borderRadius:10}}>
+                        <TextInput style={{padding: 10, fontSize: 13, width: "90%", height: 80 }} multiline  placeholder = "Ask your query, by addressing your problem clearly!" />
+                        
+                    </View>
```
Comment (shehand) [inline app/components/ForumBox/ForumBox.js:17]: Remove additional spaces. They make no sense here.

Comment (shehand) [reply app/components/ForumBox/ForumBox.js:17]: Or just use a formatter for code formatting, That would also help you to remove and organize your code more readable manner

**Review by shehand — CHANGES_REQUESTED** [review]

**Comment by shehand** [comment]
Update the PR template also. You can refer a PR that is already closed in this repo.

File: app/components/ForumBox/ForumBox.js (hunk 1 above)
Comment (AbhiKul10) [reply app/components/ForumBox/ForumBox.js]: @shehand Please review

File: App.js [hunk 2]
```diff
@@ -63,7 +65,11 @@ const AppStack = createBottomTabNavigator(
         const { routeName } = navigation.state;
         let iconName;
         if (routeName === "Home") {
-          iconName = "home";
```
Comment (shehand) [inline App.js:66]: Why replacing "home" icon with a new one?

Comment (AbhiKul10) [reply App.js:66]: fixed it 👍🏻

**Review by shehand — APPROVED** [review]
LGTM!

**Review by Wathsara — APPROVED** [review]

**Pull request merged**